src/main/common/__utils/sessionSetup.ts

```diff
@@ -0,0 +1,26 @@
+import { SessionKey } from "@companieshouse/node-session-handler/lib/session/keys/SessionKey";
+import { SignInInfoKeys } from "@companieshouse/node-session-handler/lib/session/keys/SignInInfoKeys";
+import { UserProfileKeys } from "@companieshouse/node-session-handler//lib/session/keys/UserProfileKeys";
+import { IAccessToken, ISignInInfo } from "@companieshouse/node-session-handler/lib/session/model/SessionInterfaces";
+import { AccessTokenKeys } from "@companieshouse/node-session-handler/lib/session/keys/AccessTokenKeys";
+import { Session } from "@companieshouse/node-session-handler";
+
+export const userMail = "userWithPermission@ch.gov.uk";
```

Comment:
these values can be moved to environment file or something

Reply:
These are just for mock data so can stay in this file.